src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -392,7 +395,7 @@ export default function ConfigPanel(): JSX.Element {
 
             <div className="config-panel__list">
               {formErrors
```

Comment:
I think this should also have `.some(...` or at the very least check that this isn't an empty array.